notifier.py

```diff
@@ -55,11 +60,20 @@
 
 # If you want text notifications, you'll need to have a Twilio account set up (Free Trial is fine)
 # Both of these numbers should be strings, in the format '+11234567890' (Not that it includes country code)
+if USE_TWILIO:
 twilioToNumber = getenv('TWILIOTONUM') 
```

Comment:
indentation

Reply:
I'm actually taking care of that now locally :) I was also doing some testing, looks like the nvidia_get is returning 504s from the API - it might have been locked down?